### ISSUE TITLE
Thanking the user for doing the survey. 

### ISSUE THREAD

**Issue by MichaelT4646** [MichaelT4646/Group8#5]
I noticed that on line 291 of Survey.Java we say System.out.println("Thank you again for taking our survey! We hope to see you again! :)");. However we also thank the user on line 220 System.out.println("Thank you for taking the time to complete this survey! It means a lot to us!");. I don't think it is necessary to thank the user twice. My recommendation for this would be to delete line 220 and just thank the user once all of the stats are listed.

**Comment by Andrew231456** [comment]
I do agree that there should be one line thanking the user for the survey, didn't even notice that on line 220 till you pointed it out. Nice Michael!

**Comment by itsteele** [comment]
I agree that the user should be thanked for doing the survey.  I think it's funny how none of us thought of this!
